src/components/home/Home.jsx

```diff
@@ -20,18 +28,22 @@ function Home() {
 
   const handleDateChange = (date) => {
     setSelectedDate(date);
-  }
+  };
 
   const handleClick = (event) => {
     setShow(!show);
     setTarget(event.target);
-  }
+  };
 
   return (
     <Fragment>
+      <HeaderOne></HeaderOne>
```

Comment:
In this PR, let's keep the header out of Home. You obviously have to put it there for testing, but your PR shouldn't have it there. Let Herzyl import the header into Home in one of her PRs since she's the one designing Home.